views/site/about.ejs

```diff
@@ -0,0 +1,7 @@
+<% include ../partials/header.ejs %>
+
+<h1>About The Daily Planet</h1>
+
+<p>The Daily Planet is _THE_ safest newspaper in the world, since we (entirely unknowingly) employ Superman.</p>
```

Comment:
another way of emphasizing THE is to use strong tags <p>This is a <strong>sentence</strong></p>